.github/workflows/build-and-push.yml

```diff
@@ -0,0 +1,110 @@
+# GitHub authenticates as you when using ${{ secrets.GITHUB_TOKEN }}.
+# Improve this workflow security with a GitHub PAT. Make one with the "Note" as GHCR_LINGUA_PAT here:
+# https://github.com/settings/tokens
+# then check the following permissions:
+#  repo
+#   ...
+#  workflow
+#  write:packages
+#   ...
+# Click generate, <copy to clipboard>
+# You just created a PAT
+# visit linguacafe repo, Settings -> Secrets & Variables -> Actions -> Repo Secrets -> New
+# Name: GHCR_LINGUA_PAT; Secret: <paste from clipboard>
+# update this workflow and change ${{ secrets.GITHUB_TOKEN }} to ${{ secrets.GHCR_LINGUA_PAT }}
+
+name: Build and Publish Docker images
+
+# If you want to push images on every git push to the repo:main branch
+on:
+  push:
+    branches: main
+
+## If you want to push images only on a new tag/release
+# on:
+#   release:
+#     types: [created]
```

Comment:
I suggest to comment the "on push" block and uncomment the "on release" one. Users should generally not run code from main and stick to handpicked releases.

Reply:
Main is only going to be used for final new version releases.